version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "3.12.0-SNAPSHOT"
+version in ThisBuild := "3.13.0-SNAPSHOT"
```

Comment:
Actually here's a more compelling reason: we're dropping support for scalaz-stream 0.7